src/lib/journey/stages/webauthn.svelte

```diff
@@ -92,10 +104,11 @@
     </div>
   {/if}
 
+  {#if askedForDeviceName}
```

Comment:
Still unsure about this condition. We want to show the spinner when asking for device name? In the Chromatic snapshots, it seems we removed the spinner.